### PR TITLE
Hotfix for OpenCG lattice outer attribute

### PR DESCRIPTION
**Pull request by wbinventor**

### CODE REVIEW

**Comment by wbinventor** [comment]
This PR also fixes a bug in the OpenCG compatibility module (`get_opencg_cell(openmc_cell)` in particular) that appears if and when a cell does not contain a region (or any bounding surfaces). In this case `Cell.region == None` and the current implementation of the compatibility module will throw an exception. These cases are now dealt with appropriately and an OpenCG `Cell` is returned without any bounding surfaces.

File: openmc/opencg_compatible.py [hunk 1]
```diff
@@ -909,6 +912,8 @@ def get_opencg_lattice(openmc_lattice):
     opencg_lattice.dimension = dimension
     opencg_lattice.width = pitch
     opencg_lattice.universes = universe_array
+    if outer != None:
```
Comment (paulromano) [inline openmc/opencg_compatible.py]: `if outer is not None`

File: openmc/opencg_compatible.py [hunk 2]
```diff
@@ -985,6 +991,8 @@ def get_openmc_lattice(opencg_lattice):
     openmc_lattice.pitch = width
     openmc_lattice.universes = universe_array
     openmc_lattice.lower_left = lower_left
+    if outer != None:
```
Comment (paulromano) [inline openmc/opencg_compatible.py]: `if outer is not None`

File: openmc/opencg_compatible.py [hunk 3]
```diff
@@ -486,20 +486,22 @@ def get_opencg_cell(openmc_cell):
...
-        halfspace = -1 if region.side == '-' else 1
-        opencg_cell.add_surface(get_opencg_surface(surface), halfspace)
-    elif isinstance(region, Intersection):
-        for node in region.nodes:
-            if not isinstance(node, Halfspace):
-                raise NotImplementedError("Complex cells not yet supported "
-                                          "in OpenCG.")
-            surface = node.surface
-            halfspace = -1 if node.side == '-' else 1
+    if region != None:
```
Comment (paulromano) [inline openmc/opencg_compatible.py]: Oops, missed this one too. Also should use `is not None` here.

**Pull request merged**